src/config.js

```diff
@@ -19,9 +19,7 @@ export default {
     title: null,
     body: null
   },
-  // VUE_APP_OIDC_ENABLED is not set in production. It can be set during local
-  // development to facilitate work on SSO.
-  oidcEnabled: process.env.VUE_APP_OIDC_ENABLED === 'true',
```

Comment:
I think it's the right time to stop using this environment variable. There are now two ways to run Frontend in development: `npm run dev` and `npm run dev:build`. I didn't want to multiply that by two for OIDC vs. not. I've been planning to go in this direction anyway: see #1038.